README.md

```diff
@@ -18,4 +25,17 @@ The top menu has a "Region" button which leads the user into a region comparison
 
 The intent of this app is for users to be able to explore both global trends as well as region by region comparisons in an iterative manner.    
 
+## Get Involved
+
+We are always looking for feedback and contributors! 
+
+In second milestone, the main comparison functionality of the "Region" tab was incorporated into landing page to limit the scope of having to build two tabs. Potential functionality to add for the future include:
+
+- ability to select countries on the map by clicking as a more intuitive alternative than typing in the country field
```

Comment:
Capitalise the first letter of each sentence

Reply:
completed